bentoml/deployment/store.py

```diff
@@ -146,17 +156,40 @@ def delete(self, name, namespace):
                     "Deployment '%s' in namespace: '%s' is not found" % name, namespace
                 )
 
-    def list(self, namespace, filter_str=None, labels=None, offset=None, limit=None):
+    def list(
+        self,
+        namespace,
+        operator=None,
+        labels=None,
+        offset=None,
+        limit=None,
+        order_by=None,
+        ascending_order=None,
```

Comment:
This works but I believe you want the default value to be False? `ascending_order = False`